examples/image-classification-web/src/model/normalizer.rs

```diff
@@ -0,0 +1,38 @@
+use burn::tensor::{backend::Backend, Tensor};
+
+// Values are taken from the [ONNX SqueezeNet]
+// (https://github.com/onnx/models/tree/main/vision/classification/squeezenet#preprocessing)
+const MEAN: [f32; 3] = [0.485, 0.456, 0.406];
+const STD: [f32; 3] = [0.229, 0.224, 0.225];
+
+/// Normalizer for the imagenet dataset.
+pub struct Normalizer<B: Backend> {
+    pub mean: Tensor<B, 4>,
+    pub std: Tensor<B, 4>,
+}
+
+impl<B: Backend> Normalizer<B> {
+    /// Creates a new normalizer.
+    pub fn new() -> Self {
+        let mean = Tensor::from_floats(MEAN).reshape([1, 3, 1, 1]);
+        let std = Tensor::from_floats(STD).reshape([1, 3, 1, 1]);
+        Self { mean, std }
+    }
+
+    /// Normalizes the input image according to the imagenet dataset.
+    ///
+    /// The input image should be in the range [0, 1].
+    /// The output image will be in the range [-1, 1].
+    ///
+    /// The normalization is done according to the following formula:
+    /// `input = (input - mean) / std`
+    pub fn normalize(&self, input: Tensor<B, 4>) -> Tensor<B, 4> {
+        input - self.mean.clone() / self.std.clone()
```

Comment:
Not sure about operation priority, but maybe we could put the parenthesis for clarity?

Reply:
It does look wrong, / has priority over -